shared/actions/teams.tsx

```diff
@@ -451,32 +454,22 @@ function* removePendingInvite(
   action: TeamsGen.RemovePendingInvitePayload,
   logger: Saga.SagaLogger
 ) {
-  const {teamID, username, email, inviteID} = action.payload
-  // Disallow call with any pair of username, email, and ID to avoid black-bar
-  // errors.
-  if ((!!username && !!email) || (!!username && !!inviteID) || (!!email && !!inviteID)) {
-    const errMsg = 'Supplied more than one form of identification to removePendingInvite'
-    logger.error(errMsg)
-    throw new Error(errMsg)
-  }
-
+  const {teamID, inviteID} = action.payload
   try {
     yield RPCTypes.teamsTeamRemoveMemberRpcPromise(
       {
-        allowInaction: true,
-        email: email ?? '',
-        inviteID: inviteID ?? '',
+        member: {
+          inviteid: {
```

Comment:
these types are oddly nested, but maybe that's AVDL's fault

Reply:
It's a variant, but if there's a better typescripty thing with unions we could update the avdl to TS compiler